master/pkg/searcher/operations.go

```diff
@@ -104,76 +102,31 @@ type Requested interface {
 	GetRequestID() model.RequestID
 }
 
-// Create a new trial for the search method.
-type Create struct {
+// LegacyCreate a new trial for the search method.
+type LegacyCreate struct {
```

Comment:
delete this?  It seems unused...

Reply:
yeah, i kept this mainly for reference as i was rewriting logic. deleted.